.travis.yml

```diff
@@ -4,44 +4,58 @@
 language: generic
 
 os: osx
-osx_image: beta-xcode6.1
+osx_image: xcode6.4
 
 env:
   matrix:
     
     - CONDA_PERL=5.20.3.1
+    - CONDA_PERL=5.22.0.1
+    - CONDA_PERL=5.22.2.1
```

Comment:
See that we are getting a matrix of `perl` versions here. This is undesirable as this package doesn't use `perl` at run time. It only needs it as part of the build.